Components/Atoms/GoalListItem.js

```diff
@@ -113,47 +118,60 @@ const styles = StyleSheet.create({
     },
 
 
-    box: {
-        shadowOffset:{  width: 2,  height: 5,  },
-        shadowColor: '#D1CDC7',
-        shadowOpacity: .5,
-        backgroundColor: '#EEEEEE',
-        height: 120,
-        width: width *.90,
+    neomorph: {
+        shadowOpacity: 0.7,  // <- and this or yours opacity
+        shadowRadius: 30,
         borderRadius: 30,
         borderTopColor: '#FFF',
         borderLeftColor: '#FFF',
         borderBottomColor: '#D1CDC7',
         borderRightColor: '#D1CDC7',
-        fontWeight: 'bold',
-        alignSelf: 'center',
-        borderWidth: 5,
-        justifyContent: 'center',
+        backgroundColor: '#ECF0F3', 
+        width: 300,
```

Comment:
I think it'll look better with a bit more width. We should also make this property a variable so that it can accomadate to different screen sizes (incl. iPads). See below for what I'd enter here instead of 300

`    width: width * 0.85,
`